HR/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "category": "HR",
+    "description": "HR documents, resumes, etc",
+    "display_name": "HR and people documents, such as resume",
+    "featured": {
+      "primary": "nlp_resume",
```

Comment:
this needs to get changed to `resume` since we changed the file names.

Reply:
Oh, good catch, doing that now